app/controllers/hq/users_controller.rb

```diff
@@ -6,4 +6,24 @@ def index
     @users = User.all
   end
 
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(user_params)
```

Comment:
If you extract out `User.find` you could combine lines 14-15 here `if method.update_attr....`